test/test_transformers.py

```diff
@@ -1503,8 +1504,9 @@ def test_nested_fails_on_padding_head_dim(self, device):
 
 
     @onlyCUDA
-    @unittest.skipIf(not PLATFORM_SUPPORTS_FUSED_ATTENTION or not isSM5xDevice, "Does not support fused SDPA or not SM50 hardware")
-    def test_mem_efficient_fail_bfloat16_sm50(self, device):
+    @unittest.skipIf(not PLATFORM_SUPPORTS_FUSED_ATTENTION or not isLessThanSM80Device,
+                     "Does not support fused SDPA or not less than sm80 hardware")
```

Comment:
This is a hard to parse comment, maybe more: 
"Current platform does not support fused SDPA or is an SM80+ device."